src/EPR.RegulatorService.Frontend.Web/ViewComponents/RegistrationSubmissions/CompliancePaymentDetailsViewComponent.cs

```diff
@@ -0,0 +1,40 @@
+using System.Diagnostics.CodeAnalysis;
+using System.Threading.Tasks;
+
+using EPR.RegulatorService.Frontend.Core.Extensions;
+using EPR.RegulatorService.Frontend.Core.Models.RegistrationSubmissions;
+using EPR.RegulatorService.Frontend.Core.Services;
+using EPR.RegulatorService.Frontend.Web.ViewModels.RegistrationSubmissions;
+
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.ViewComponents;
+
+namespace EPR.RegulatorService.Frontend.Web.ViewComponents.RegistrationSubmissions;
+
+public class CompliancePaymentDetailsViewComponent(IPaymentFacadeService paymentFacadeService, ILogger<CompliancePaymentDetailsViewComponent> logger) : ViewComponent
+{
+    public async Task<ViewViewComponentResult> InvokeAsync(RegistrationSubmissionDetailsViewModel viewModel)
+    {
+        try
+        {
+            var model = await paymentFacadeService.GetCompliancePaymentDetailsAsync(new CompliancePaymentRequest
+            {
```

Comment:
The request will be updated as part of https://dev.azure.com/defragovuk/RWD-CPR-EPR4P-ADO/_workitems/edit/477227